workspaces/src/types.rs

```diff
@@ -2,11 +2,16 @@
 /// and internal libraries like near-jsonrpc-client requires specific versions
 /// of these types which shouldn't be exposed either.
 use std::convert::TryFrom;
+use std::fmt;
 use std::path::Path;
 
 pub use near_account_id::AccountId;
 pub(crate) use near_crypto::{KeyType, Signer};
-use near_primitives::serialize::from_base;
+pub use near_primitives::views::ExecutionStatusView;
```

Comment:
Seems not ideal to expose this type from primitives. There are implementations and details of this that might be tricky to maintain, and the types within this enum do not match anything from this lib

Reply:
Yup, thanks for catching that. Thought this was cleaned up when I removed all the exports but guess not